src/foam/u2/view/ReciprocalSearch.js

```diff
@@ -99,11 +111,17 @@ foam.CLASS({
     },
     {
       class: 'Int',
-      name: 'selectedCount'
+      name: 'selectedCount',
+      postSet: function() {
```

Comment:
Can we get rid of these postSets and change `isLoading` boolean to a `loadingRequests` int.
The in updateTotalCount and updateSelectedAccount, they do a loadingRequests++ before their select and a loadingRequests-- after, then you check if loadingRequests == 0 when deciding what to show?

Reply:
Resolved by 24246fe6f5c09eb6b84f47ee279e11d36cfc8205.